definitions/features/instance.rb

```diff
@@ -62,6 +62,8 @@ def ping
 
   def server_connection
     net = Net::HTTP.new(ForemanMaintain.config.foreman_url, ForemanMaintain.config.foreman_port)
+    # the foreman_ping doesn't work without VERIFY_NONE
+    net.verify_mode = OpenSSL::SSL::VERIFY_NONE
```

Comment:
In general I'm not too happy about `VERIFY_NONE` use. Perhaps you can get the foreman_ca_cert from the installer's answer files to verify the connection?